### PR TITLE
[Instrument List] Update test plan

### PR DESCRIPTION
**Pull request by miebeers**
Added item in the test plan about access to the module, and clarified wording to make the test plan more readable.

### CODE REVIEW

File: modules/instrument_list/test/TestPlan.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+5. Verify that `Double Data Entry` is available if it is enabled for that instrument, no link if not enabled
+6. Verify that the meta data for the candidate/visit is correct - eg DoB, Gender, Visit Label, Visit Status, etc
+7. Verify that BVL QC flags (`BVL QC Type` and `BVL QC Status`) are working and only available if the user has the `Behavioural QC` permission
+8. Verify that `Stage` can only be set after Data Entry (and DDE if applicable) is complete, and conflicts are resolved
+9. Verify that if the current `Stage` is `Not Started`, `Start Next Stage` is available (provided that it's a valid Timepoint)
+10. Verify that `Send to DCC` is only available after Visit is marked as Pass/Failure/Withdrawal, and if the user has the `Send to DCC` permission
+11. Verify that `Reverse Send from DCC` permission is necessary to unsend to DCC
+12. Verify that breadcrumbs work
+13. Verify that values are correct for `Within Optimal` visit window and `Within Permitted` visit window. The optimal and permitted visit window values can be found in the `Visit_Windows` table of the database.
```
Comment (PapillonMcGill) [inline modules/instrument_list/test/TestPlan.md]: Only item 13 as en ending dot. Add every where or remove every where for consistency.

File: modules/instrument_list/test/TestPlan.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-11. Verify that `Reverse Send from DCC` permission is necessary to unsend to DCC
-12. Verify that breadcrumbs work
-13. Verify that values are correct for `Within Optimal` visit window and `Within Permitted` visit window. The optimal and permitted visit window values can be found in the `Visit_Windows` table of the database.  
+1. Verify that in order to access the module, the user must either have the `Across all sites access candidate profiles` permission, or be at the same site as the visit, or at the same site as one of the candidate's other visits.
+2. Verify that in order to update the current `Stage` status, the user requires `Data entry` permissions, and must be at the proper site. Exception: if the current `Stage` status is `Approval`, then only the `Behavioural QC` permission is necessary to update the `Stage` status.
+3. Verify that each instrument in the list directs to the corresponding instrument page.
+4. Verify that the flags' values (Data Entry, Administration, Feedback, Double Data Entry Status) for each instrument in the list are showing up appropriately.
```
Comment (PapillonMcGill) [inline modules/instrument_list/test/TestPlan.md]: Does `Status` part of DDE or apply to all of theses values?

File: modules/instrument_list/test/TestPlan.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+5. Verify that `Double Data Entry` is available if it is enabled for that instrument, and that there is no link if it is not enabled.
+6. Verify that the meta data for the candidate/visit is correct - eg DoB, Gender, Visit Label, Visit Status, etc.
```
Comment (PapillonMcGill) [inline modules/instrument_list/test/TestPlan.md]: candidate and visit would be better IMO

File: modules/instrument_list/test/TestPlan.md (hunk 2 above)
Comment (PapillonMcGill) [reply modules/instrument_list/test/TestPlan.md]: Ok, thanks

File: modules/instrument_list/test/TestPlan.md (hunk 3 above)
Comment (miebeers) [reply modules/instrument_list/test/TestPlan.md]: I disagree! Its the combination of the candidate and visit... the slash was there before, but it makes sense to me. The candidate dictates the DoB and gender, the visit label and visit status apply to the visit. The combination of candidate/visit creates that unique set of metadata.

Comment (miebeers) [reply modules/instrument_list/test/TestPlan.md]: Actually it should probably be candidate/timepoint.

File: modules/instrument_list/test/TestPlan.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Instrument List Test Plan
```
Comment (johnsaigle) [inline modules/instrument_list/test/TestPlan.md]: Could you add a `#` at the front of this so it shows up as a big header in the markdown file?

**Pull request merged**